charts/kube-prometheus-stack/Chart.yaml

```
@@ -22,7 +22,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 39.11.0
+version: 39.11.1
```

```suggestion
version: 39.13.2
```

please merge after: https://github.com/prometheus-community/helm-charts/pull/2430#pullrequestreview-1105487003 which has version 39.13.1

---

Done